src/home_robot/home_robot/mapping/semantic/categorical_2d_semantic_map_module.py

```diff
@@ -86,15 +97,21 @@ def __init__(
         self.local_map_size = self.local_map_size_cm // self.resolution
         self.xy_resolution = self.z_resolution = map_resolution
```

Comment:
There's a bunch of divisions here; maybe add defaults/check if anything is 0 and raise error?

Reply:
these are all presets and this is from devendra's code, would rather not change anything